site/docs-md/cordova/using-cordova-plugins.md

```diff
@@ -0,0 +1,51 @@
+# Using Cordova Plugins and Ionic Native
+
+When developing an app that uses Capacitor, it's possible to use both Cordova and Ionic Native plugins.
+
+## Installing Cordova Plugins
+
+Simply install your plugin of choice, sync your project, finish any required native project configuration, and you're ready to go:
+
+```bash
+npm install cordova-plugin-name
+npx cap sync
+```
+
+## Updating Cordova Plugins
+
+Similar to the installation steps. Capacitor will detect the new version of the plugin and install it.
+
+```bash
+npm install cordova-plugin-name
+npx cap update
+```
+
+## Installing Ionic Native Plugins
+[Ionic Native](https://ionicframework.com/docs/native) provides TypeScript wrappers and a consistent API and naming convention for easier development with Cordova plugins. It's supported in Capacitor, so whenever you find an Ionic Native wrapper you'd like to use, install the JavaScript code, install the corresponding Cordova plugin, then sync your project:
+
+```bash
+npm install @ionic-native/javascript-package-name
+cordova plugin add cordova-plugin-name
+npx cap sync
+```
+
+## Updating Ionic Native Plugins
+
+Similiar to the installation steps. Update the Ionic Native JavaScript library, remove then re-add the Cordova plugin, then update your project:
+
+```bash
+npm install @ionic-native/javascript-package-name
+cordova plugin remove cordova-plugin-name
```

Comment:
same, use npm install/npm uninstall instead of cordova command

probably also good to tell them to uninstall the wrapper first as the section is about updating, so they probably have the old version installed and npm might not get a newer one on npm install